source/_integrations/matter.markdown

```diff
@@ -255,6 +255,32 @@ Follow these steps if you have added a Matter device to Home Assistant and you w
    - For example, if you shared it with Google Home, it lists **Google LLC**.
      ![image](/images/integrations/matter/matter_share-device-with-other-platform.png)
 
+## About Matter device information
+
+The device information section provides some networking information of a device. For devices that are behind a bridge, this information is not available.
+
+1. To view the device details, go to {% my integrations title="**Settings** > **Devices & services**" %} and on the **Matter (BETA)** integration card, select **Devices**.
+2. From the list of devices, select the device you want to look at.
+
+<p class='img'>
+<img src='/images/integrations/matter/matter_device-info.png' alt='Matter device information'>
+</p>
+
+This section provides a bit more information on some of the categories:
+
+**Network type**: Shows which network type the device belongs to. For example, Thread or Wi-Fi.
+
+**Device type**: There are 4 device types:
+
+- **End device**: End device that always keeps it radio on. Typically these devices are not battery-powered.
+- **Sleepy end device**: non-routing end device. Normally disabled, wakes on occasion to poll for messages from its parent. Typically battery-powered.
+- **Routing end device**: End device that always keeps it radio on and can route traffic for other, non-routing end devices.
+- **Bridge**
+
+**Network name**: Name of the network the device joined when it was first commissioned first.
+
+**IP addresses**: For Thread networks, typically more than one IPv6 address is shown: link local, unique local, and global unicast. Sometimes, no IPv6 address is shown.
```

Comment:
```suggestion
**IP addresses**: Typically more than one IPv6 address is shown: link local, unique local, and global unicast. In some cases a device also supports IPv4. In that case there will also be listed an IPv4 address here.
```

the missing IP address on the P2 sensor is a bug actually

Reply:
so you want to explain all the found ip addresses ? You could also shortcut it to: the IP adresses reported/used by this device.